### PR TITLE
GPII-2951- WIP - Utility to export the records of a Route53 zone

### PR DESCRIPTION
**Pull request by amatas**
Here it is a small utility to save the A,CNAME and TXT records in to a TF file, plus the actual registries of the gpii.net zone.

Related tickets:
GPII-2951
GPII-3199

### CODE REVIEW

**Comment by mrtyler** [comment]
LGTM, (though this is WIP so I didn't really review the code).

Your branch name says NOJIRA but there is a Jira for this.

File: utils/gpii-net.tf [hunk 1]
```diff
@@ -0,0 +1,717 @@
+### A ###
```
Comment (natarajaya) [inline utils/gpii-net.tf]: Should not this be `aws/utils`, since it is dealing with AWS resources?

**Comment by natarajaya** [comment]
LGTM.

File: utils/gpii-net.tf (hunk 1 above)
Comment (amatas) [reply utils/gpii-net.tf]: Right, I made this utility before the new repository structure

**Comment by amatas** [comment]
@stepanstipl , it stills relevant because we're planning to move the DNS zones from Route53 to GCP DNS

**Pull request closed without merging**